core/src/main/java/fr/sncf/osrd/train/StandaloneTrainSchedule.java

```diff
@@ -13,14 +13,14 @@
 
     public final ArrayList<TrainStop> stops;
 
-    public final ArrayList<? extends Allowance> allowances;
+    public final ArrayList<HardenedMarecoAllowance> allowances;
```

Comment:
What is the reason for this change? Do we specifically want Mareco here and no other range? (if we eventually have more)

Reply:
Nope i guess i just did something wrong here